kie-drools-wb/kie-drools-wb-webapp/src/main/java/org/kie/workbench/drools/backend/server/AppSetup.java

```diff
@@ -76,34 +70,11 @@ public AppSetup( @Named("ioStrategy") final IOService ioService,
     }
 
     @PostConstruct
-    public void assertPlayground() {
+    public void init() {
         try {
             configurationService.startBatch();
-            final String exampleRepositoriesRoot = System.getProperty( "org.kie.example.repositories" );
```

Comment:
If `org.kie.example.repositories` and `org.kie.example` are going to be removed (and not reintroduced somewhere else) please make sure, that it's also removed from kie-docs to keep it in sync